scripts/check_suitespec_coverage.py

```diff
@@ -39,21 +42,24 @@ def unmatched() -> set[str]:
 unmatched_patterns = unmatched()
 
 if uncovered_sources:
-    print("Source files not covered by any suite specs:", len(uncovered_sources))
+    print(f"▶️ {len(uncovered_sources)} source files not covered by any suite specs:")
     for f in sorted(uncovered_sources):
-        print(f"  {f}")
+        print(f"    {f}")
+    print()
 if uncovered_tests:
-    print("Test scripts not covered by any suite specs:", len(uncovered_tests))
+    print(f"🧪 {len(uncovered_tests)} test files not covered by any suite specs:")
     for f in sorted(uncovered_tests):
-        print(f"  {f}")
+        print(f"    {f}")
+    print()
 if not uncovered_sources and not uncovered_tests:
-    print("All files are covered by suite specs")
+    print("✨ 🍰 ✨ All files are covered by suite specs")
 
 if unmatched_patterns:
-    print("Unmatched patterns:", len(unmatched_patterns))
+    print(f"🧹 {len(unmatched_patterns)} unmatched patterns:")
     for p in sorted(unmatched_patterns):
-        print(f"  {p}")
+        print(f"    {p}")
+    print()
 else:
-    print("All patterns are matching")
+    print("✨ 🧹 ✨ All patterns are matching")
```

Comment:
Shouldn't this be cake too?

Reply:
This is more like a linting step, so ✨ 🧹 ✨  all clean